kernel/libs/aster-bigtcp/src/iface/poll.rs

```diff
@@ -507,6 +525,11 @@ impl<E: Ext> PollContext<'_, E> {
             }
         }
 
+        // Since we are working with UDP sockets,
+        // the `dead_conns` collection only includes dead TCP connections.
+        // Therefore, `dead_conns` should be empty.
```

Comment:
I don't think `Since A, B. Therefore, C` means `A + B => C`.

Maybe you want something like `C, because A and B`?